packages/common/tests/data/energyWebChain.json

```diff
@@ -0,0 +1,79 @@
+{
+  "name": "energyWebChain",
```

Comment:
Awesome test data 😄

Reply:
This is taken from #968 😄 , I would actually want to wait for a quick ok from @matextrem the author until we merge here.